osquery/tables/events/windows/windows_events.cpp

```diff
@@ -42,7 +42,12 @@ class WindowsEventSubscriber
  public:
   Status init() override {
     auto wc = createSubscriptionContext();
-    for (const auto& chan : osquery::split(FLAGS_windows_event_channels, ",")) {
+    for (auto& chan : osquery::split(FLAGS_windows_event_channels, ",")) {
+      // We remove quotes if they exist
+      auto quoteLoc = chan.find("\"");
+      if (quoteLoc != std::string::npos) {
+        chan.erase(quoteLoc);
```

Comment:
Are any of the things below valid?
`--windows_event_channels=System,Security,"Windows PowerShell"`
`--windows_event_channels='System,Security,Windows PowerShell'`
`--windows_event_channels=System,Security,'Windows PowerShell'`
`--windows_event_channels="System,Security,'Windows PowerShell'"`